tests/framework/AnalyticModels/projectile.py

```diff
@@ -86,7 +99,7 @@ def run(raven, inputs):
   vars = {'x0': get_from_raven(raven,'x0', 0),
           'y0': get_from_raven(raven,'y0', 0),
           'v0': get_from_raven(raven,'v0', 1),
-          'ang': get_from_raven(raven,'v0', 45),
+          'ang': get_from_raven(raven,'angle', 45),
```

Comment:
this change is fixing a bug in this model, and will affect any test using the projectile model.

Reply:
@dylanjm this affects the projectile model in `raven/tests/framework/AnalyticTests/`, not the one in the workshop exercises (I think). It may be worth pointing the workshop exercises to the "correct" one, possibly.

Reply:
FWIW I would like to remove other versions of projectile and always point to the same model